### DISCUSSION TITLE
Gamified-PKM 0.0.87

### DISCUSSION THREAD

**Discussion by saertna** [saertna/obsidian-gamified-pkm#59] [General]
## Overview of Elements
Majurity Level of notes
Levels
Status Points
Progress bar
Badges
Challenges
Profile Page
Booster
crafting booster with ingredients

## What's Changed
* Check for new version by @saertna in https://github.com/saertna/obsidian-gamified-pkm/pull/58

**Full Changelog**: https://github.com/saertna/obsidian-gamified-pkm/compare/0.0.86...0.0.87

<hr /><em>This discussion was created from the release <a href='https://github.com/saertna/obsidian-gamified-pkm/releases/tag/0.0.87'>Gamified-PKM 0.0.87</a>.</em>

**Comment by soge-king** [comment]
Hi, im curious as to how to edit/modify the setting so that i can sort of do a similar Statblock like DnD(INT,CON,STR,DEX,CHAR,WIS), and each task would give me xp/lvl up an "ability"/make daily tasks that improve a stat/etc. i also want to try and integrate the Rewarded aspect.  like lvling up or compleating a task gives you a common/rare/epic/legendary reward (have a snack/take a nap/play a agme etc)

**Comment by soge-king** [comment]
also, just some ideas.
1 - Preset List of lvl names related to chosen class (class is arbitrary, but does give a better feel for progression. in the next comment ill show what i added to my avatar page)
2- Relate Specific folder and/or notes to a certain stat or skill. the xp for a stat upgrade should be a lot higher and correlate with the skill asociated with it (exp.: 
Task/Note/File ->Skill/Ability -> STAT
Cardio (in daily notes) -> Sprinting -> CON
Updating a note on a math subject -> Mathematics -> INT)
3 - set maturaty lvls to a selected number/ this can correlate with how hard/easy a task or subject is
4 - select a photo/icon to represent the "player" i.e. you
5 - a selectable sound byte per completed tasks/lvl up. this i just like because it feels nice to hear a "ding" when you finish some. could be my  ADHD, but whatever.

**Comment by soge-king** [comment]
i like artificers in DnD so i went with that theme:
Level 5: *Novice Artificer*
level 10: *Experienced Initiate*
level 20: *Skilled Artificer*
level 30: *Expert Artificer*
level 40: *Mechanist Initiate*
level 50: *Skilled Mechanist Initiate*
level 60: *Expert Mechanist*
level 70: *Arcanoengineer*
level 80: *Senior Arcanoengineer*
level 90: *Arch Arcanoenginner*
level 99: *Grand Magnus*
level 100: *Machine God*

**Comment by saertna** [comment]
Hey @soge-king, please have a look  at https://github.com/saertna/obsidian-gamified-pkm/discussions/60#discussioncomment-8326100 in which I replayed to your ideas.
